contracts/sg721-updatable/src/contract.rs

```diff
@@ -295,19 +316,8 @@ pub fn migrate(deps: DepsMut, _env: Env, _msg: Empty) -> Result<Response, Contra
     }
 
     FROZEN_TOKEN_METADATA.save(deps.storage, &false)?;
-
-    let mut collection_info = COLLECTION_INFO.load(deps.storage)?;
-    let current_royalty_info = collection_info.royalty_info;
-    if let Some(royalty_info) = current_royalty_info {
-        let new_royalty_info = RoyaltyInfo {
-            payment_address: royalty_info.payment_address,
-            share: royalty_info.share,
-            // setting updated_at to 1 day ago to make updating royalties possible right after a migration
-            updated_at: Some(_env.block.time.minus_seconds(86400)),
-        };
-        collection_info.royalty_info = Some(new_royalty_info);
-        COLLECTION_INFO.save(deps.storage, &collection_info)?;
-    }
+    // setting updated_at to 1 day ago to make updating royalties possible right after a migration
+    ROYALTY_UPDATED_AT.save(deps.storage, &_env.block.time.minus_seconds(86400))?;
```

Comment:
rename the variable to `env` usually `_` is for unused variables but when you switch to use it ideally you rename the param in the method.